From b4e383c70f819d33a2dbaef2a06f1e1d8b07bb78 Mon Sep 17 00:00:00 2001
From: Damien Caliste <dcaliste@free.fr>
Date: Tue, 11 Dec 2018 13:27:28 +0100
Subject: [PATCH] Reenable transparent background.

---
 words/part/KWCanvasItem.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/words/part/KWCanvasItem.cpp b/words/part/KWCanvasItem.cpp
index 4f2f9da21e7..7e7e265be5b 100644
--- a/words/part/KWCanvasItem.cpp
+++ b/words/part/KWCanvasItem.cpp
@@ -162,7 +162,7 @@ void KWCanvasItem::updateCanvas(const QRectF &rc)
 
 void KWCanvasItem::paint(QPainter *painter, const QStyleOptionGraphicsItem *option, QWidget *)
 {
-    painter->fillRect(option->exposedRect, QColor::fromRgb(232, 233, 234));
+    // painter->fillRect(option->exposedRect, QColor::fromRgb(232, 233, 234));
     KWCanvasBase::paint(*painter, option->exposedRect);
 }
 
-- 
2.29.2

